aten/src/ATen/core/type.cpp

```diff
@@ -916,7 +916,7 @@ size_t ClassType::addAttribute(
     bool is_parameter) {
   const char* what = is_parameter ? "parameter" : "attribute";
   checkNotExist(name, what);
-  checkNoAny(*this, what, name, type);
+  // checkNoAny(*this, what, name, type);
```

Comment:
I disabled this check temporarily in order to register an `backend_handles` and `extra_infos` attributes of type `Dict[str, Any]` on the generated `ScriptModule` so as to allow backends to define what the backhand specific types look like. Is there a way to achieve this without declaring `Any` type attributes?

Reply:
Still unable to add attributes to the generated module that involve `Any` types without disabling this, but not sure if i can get rid of it altogether instead of commenting it out.

Reply:
I wonder if this is supposed to be a thing: https://circleci.com/gh/pytorch/pytorch/5302241